### ISSUE TITLE
RuntimeError: one of the variables needed for gradient computation

### ISSUE THREAD

**Issue by MuHeDing** [ADLab-AutoDrive/ICKD#2]
I have run your code and found a question:

RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation

I want to konw how to solve this questions, thank you!

**Comment by JasonCZH4** [comment]
hi,did you solve this problem?

**Comment by JasonCZH4** [comment]
> I have run your code and found a question:
> 
> RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation
> 
> I want to konw how to solve this questions, thank you!

hi,did you solve this problem?

**Comment by MuHeDing** [comment]
yes, you need to change the code of network! you can refer to the network code of CRD 




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月16日(星期三) 凌晨0:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ADLab-AutoDrive/ICKD] RuntimeError: one of the variables needed for gradient computation (Issue #2)





 
hi,did you solve this problem?

**Comment by JasonCZH4** [comment]
@MuHeDing Thank you so much!   It works!
